intl.emu

```diff
@@ -85,16 +85,17 @@ location: https://github.com/tc39/proposal-decimal/
           1. Let _sign_ be ~negative~.
           1. Set _x_ to 0.
         1. <ins>If _x_ is a Decimal128 object, then</ins>
-          1. <ins>Perform ? RequireInternalSlot(_O_, [[Decimal128Data]]).</ins>
```

Comment:
I think you still need this `?`.

Reply:
I was going to write a long comment here but deleted it I realized, while trying to respond, there there's a mistake of mine one level higher than where it appears to be: the argument to the AO under discussion should be a Decimal128 *value*, not a Decimal128 *object*. The line can be deleted.